src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -66,7 +68,15 @@ export default class OrganizationDiscover extends React.Component {
   }
 
   componentDidMount() {
-    if (this.props.savedQuery) {
+    const {savedQuery, queryBuilder} = this.props;
+
+    const keys = ['fields', 'conditions', 'aggregations', 'orderby'];
```

Comment:
Should `limit` be on this list? Also what do you think of auto-running in the case of a querystring and not otherwise? I think it more closely mimics the behaviour of discover - which updates the querystring upon successfully running a query

Reply:
Ah yeah I think that would make more sense if we only checked for query string

Reply:
Updated 776807d